crates/bevy_math/src/float_ord.rs

```diff
@@ -14,6 +17,7 @@ use std::{
 /// Wrapping a float with `FloatOrd` breaks conformance with the standard
 /// by sorting `NaN` as less than all other numbers and equal to any other `NaN`.
 #[derive(Debug, Copy, Clone)]
+#[cfg_attr(feature = "bevy_reflect", derive(Reflect), reflect(Debug))]
```

Comment:
We can also add the following it looks like:
```suggestion
#[cfg_attr(feature = "bevy_reflect", derive(Reflect), reflect(Debug, PartialEq, Hash))]
```

Reply:
Ah right, I was only looking at the derives. Not the manual, impls.